functorch/experimental/cond.py

```diff
@@ -29,15 +29,6 @@ def suspend_mode(mode):
     finally:
         torch._C._set_torch_dispatch_mode(mode)
 
-@contextmanager
-def enable_mode(mode):
```

Comment:
@voznesenskym I wasn't able to find this actually used anywhere--from grepping through pytorch and dynamo. Am I missing somewhere else?

With the mode stack change, I can't quite build the exact same thing but if I can understand what it was suppopsed to be used for, I can add it back here or later